src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java

```diff
@@ -279,12 +271,6 @@ void map_shouldReturnCorrectlyMappedResource() {
     validateTitle3(edgeIterator.next());
     validateVariantTitle(edgeIterator.next());
     validateParallelTitle(edgeIterator.next());
-    validateProviderEvent(edgeIterator.next(), PE_MANUFACTURE, "Manufacture261");
```

Comment:
It's great to have separate tests for comprehensive checks but I would still leave one option in this test to see how marc2bib conversion is working with marc file which has different fields like 245, 008, 043, 261, etc. (```full_marc.jsonl```) Because files like ```marc_261.jsonl``` only contain one certain field...

Reply:
Ok... I will leave one 264 here.